src/_tests_/NavBar.test.js

```diff
@@ -0,0 +1,8 @@
+import { render } from '@testing-library/react';
+
+import NavBar from '../components/NavBar';
+
+test('should render NavBar', () => {
+  const { container } = render(<NavBar />);

```

Comment:
You made great efforts for testing your project! However, your Navbar test is failing on my console:

<img width="405" alt="Screen Shot 2022-04-30 at 11 27 37 PM" src="https://user-images.githubusercontent.com/69011963/166118077-a54d0e8b-a92a-430b-965d-5857d84ca451.png">
<img width="1083" alt="Screen Shot 2022-04-30 at 11 28 09 PM" src="https://user-images.githubusercontent.com/69011963/166118083-75934b30-b6f4-4ecf-8298-3a838f8e7094.png">

Please try to fix this error to make it approvable!